### PR TITLE
add redactor support.

### PR DESCRIPTION
**Pull request by yariplus**

### CODE REVIEW

**Comment by yariplus** [comment]
Fixes #1

File: static/lib/main.js [hunk 1]
```diff
@@ -2,20 +2,38 @@
...
+			uploader.show({
+				title: 'Upload Audio',
+				description: 'Upload an audio file for embedding into your post',
+				route: config.relative_path + '/plugins/nodebb-plugin-audio-embed/upload'
+			}, callback);
+		});
+	}
+
 	$(window).on('action:app.load', function() {
 		require(['composer/formatting', 'composer/controls', 'uploader'], function(formatting, controls, uploader) {
```
Comment (julianlam) [inline static/lib/main.js]: No need to have uploader here, then :smile:

Comment (yariplus) [inline static/lib/main.js]: oh lol forgot to remove that

**Comment by yariplus** [comment]
Should be good now.

**Comment by julianlam** [comment]
:clap: Thank you!

**Pull request merged**